sale_product_bundle_layout/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Sale Product bundle Layout
+==========================
+
+This module was written to extend the functionality of ``sale_layout`` and
+``sale_product_bundle`` modules and allow you to set a default section on
+product bundles.
+
+
+Usage
+=====
+
+On product bundles add default section which should be defined on sale order
+lines.
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+*
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/sale-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/sale-workflow/issues/new?body=module:%20sale_product_bundle_layout%0Aversion:%201.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
version should be 8.0, it's for recognizing which of the branches is the affected one